OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -278,6 +284,16 @@ public override void Cancel(Actor self)
 			base.Cancel(self);
 		}
 
+		public override CPos GetDestination(Actor self)
+		{
+			return this.destination.Value;
+		}
+
+		public override void DrawLines(Actor self, List<Target> currentTargets)
```

Comment:
You probably want to use IEnumerable\<Target\> instead of List as that prevents the overridden methods from changing the List as Lists are passed by reference.